playwright-e2e/dsm/component/tabs/survey-data-tab.ts

```diff
@@ -0,0 +1,29 @@
+import { Locator, Page, expect } from '@playwright/test';
+import { SurveyDataPanelEnum } from './enums/survey-data-enum';
+
+export default class SurveyDataTab {
+  constructor(private readonly page: Page) {
+  }
+
+  public async getActivityData(activity: SurveyDataPanelEnum, label: string): Promise<string[]> {
+    await this.activityDataPanel(activity).waitFor({state: 'visible' });
+    const region = this.activityDataPanel(activity).locator('xpath=//*[@role="region"]');
```

Comment:
nice way to make locators neater (using a private method) + add locator method to it